mobile_config/src/gateway_info.rs

```diff
@@ -73,6 +73,17 @@ pub enum DeviceType {
     WifiDataOnly,
 }
 
+impl DeviceType {
+    fn to_sql_argument(&self) -> String {
+        match self {
+            DeviceType::Cbrs => "\"cbrs\"".to_string(),
+            DeviceType::WifiIndoor => "\"wifiIndoor\"".to_string(),
+            DeviceType::WifiOutdoor => "\"wifiOutdoor\"".to_string(),
+            DeviceType::WifiDataOnly => "\"wifiDataOnly\"".to_string(),
+        }
+    }
+}
```

Comment:
should be able to make this more general and just `impl std::fmt::Display` for this, avoid the escape characters, and it will work just as well

Reply:
I've made it more generic but still forced to use quotas to make it works